packages/core/components/QuerySidebar/Query.module.css

```diff
@@ -8,19 +8,24 @@
     background-color: var(--highlight-background-color);
 }
 
```

Comment:
Does this cover "Collapsed query state styling should be improved" addressed? It seems very similar to me.

Reply:
Yep - see the figma design here.

https://www.figma.com/design/oNQvOAK11YINcbPvNmkgwO/Public-File-Explorer?node-id=374-11533&t=fXOvHtmRv4MBWf8G-0